fms/utils/serialization.py

```diff
@@ -67,58 +110,79 @@ def list_sources(architecture: str):
     return list(__adapters[architecture].keys())
 
 
-def _legacy_attn_unfused_to_fused_adapter(orig_sd):
+def _pre006_adapter_step(
```

Comment:
I believe this needs to also include the `_mlp_glu_unfused_to_fused_adapter_step`

Reply:
only needed for llama, made the correction